docker-compose.yaml

```diff
@@ -41,13 +41,14 @@ services:
     env_file:
       - './.env'
     image: 'grafana/tempo:latest'
+    volumes:
+      - '${TEMPO_CONFIG_PATH_LOCAL}:${TEMPO_CONFIG_PATH}'
     command:
-      - '-search.enabled=true'
-      - '-storage.trace.backend=local'
       - '-storage.trace.local.path=${TEMPO_TEMP_PATH}/${TEMPO_TRACES_PATH}'
       - '-storage.trace.wal.path=${TEMPO_TEMP_PATH}/${TEMPO_WAL_PATH}'
       - '-auth.enabled=false'
```

Comment:
```suggestion
```

This can be moved to the config file and it's also deprecated. Will do suggestions in `tempo.yaml`